pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/sink/PulsarSink.java

```diff
@@ -268,7 +270,7 @@ public void open(Map<String, Object> config, SinkContext sinkContext) throws Exc
     }
 
     @Override
-    public void write(Record<T> record) throws Exception {
```

Comment:
why are removing "throws Exception" here?

Reply:
because nothing ever throws any exception in the code

Reply:
https://github.com/apache/pulsar/blob/master/pulsar-io/core/src/main/java/org/apache/pulsar/io/core/Sink.java#L44